ebusd-2.1.x/de/vaillant/_templates.csv

```diff
@@ -97,3 +97,12 @@ shortphone,STR:6,,,Telefonnummer
 mamode,UIN,0=circulation;1=dryer;2=zone;3=legiopump,,
 offmode,UIN,0=eco;1=night,,
 rcmode,UIN,0=off;1=modulating;2=thermostat,,
+slotIndex,UCH,,,Index eines Zeitslots. Start by 0 f�r den ersten Zeitslot.
```

Comment:
not sure what encoding the file has, should be UTF-8, please check

Reply:
Done. Not sure what I created before either. Is now UTF-8.